scirpy/_plotting/_base.py

```diff
@@ -289,6 +289,108 @@ def ol_scatter(
     return ax
 
 
+@_doc_params(common_doc=_common_doc)
+def volcano(
```

Comment:
I am open to discussion here. The reason I thought it is worth a separate function are the follows:  
- ol_scatter uses the third column for size and has no option colors (yet), while volcano uses that column for coloring
- All the points in a volcano are of the same size, preferably small and transparent, while the points in the overlap are usually bigger, preferably with an outline.
- Volcano is centered around zero on the x axis, while zero is always the minimum for overlaps

Reply:
> ol_scatter uses the third column for size and has no option colors (yet), while volcano uses that column for coloring

I think it would be nice being able to pass the column names for `x`, `y`, `size` and `color` anyway. This makes the functions a lot more flexible. 

Also the other two options sound like they would be easy to turn into parameters. 

But if you think there will be a lot of volcano-specific stuff added to that plotting function at some later stage, it is also fine to keep them separate.